Magento_Review/web/js/go-to-review.babel.js

```diff
@@ -0,0 +1,38 @@
+define(() => (config, element) => {
+  element.addEventListener('click', () => {
+    const element = document.getElementById(config.reviewId);
+
+    if (element) {
+      const parentElement = findAncestor(element, 'tab__content');
+      setActiveFormTab(parentElement, parentElement.dataset.content);
+    }
+  });
+
+  function findAncestor(element, containClass) {
+    while ((element = element.parentElement) && !element.classList.contains(containClass));
+    return element;
+  }
+
+  function setActiveFormTab(oneTab, searchingTab) {
+    oneTab.parentElement.querySelectorAll('.tab__title, .tab__content').forEach(tab => {
+      if (tab.dataset.content !== searchingTab && tab.dataset.tab !== searchingTab) {
+        if (tab.classList.contains('tab__title--active')) {
+          tab.classList.remove('tab__title--active');
+        }
+
+        if (tab.classList.contains('tab__content--active')) {
+          tab.classList.remove('tab__content--active');
+        }
+      }
+      else {
+        if (tab.classList.contains('tab__title') && !tab.classList.contains('tab__title--active')) {
+          tab.classList.add('tab__title--active');
+        }
+
+        if (tab.classList.contains('tab__content') && !tab.classList.contains('tab__content--active')) {
+          tab.classList.add('tab__content--active');
+        }
```

Comment:
Perhaps sth like that for adding and removing classes? (not necessary... just idea)
28-33 lines
```
addActiveClass(tab,'tab__title','tab__title--active');
addActiveClass(tab,'tab__content','tab__content--active');
```
```
function addActiveClass(el, selector, activeSelector) {
 if (el.classList.contains(selector) && !el.classList.contains(activeSelector) {
   el.classList.add(activeSelector)
}
```

Reply:
It's an ok idea but only for this case. I don't know if we have similar situations in other files but AFAIS there is not that much similar situations to this. I mean we are adding and removing classes but when we want to change it's often required to go deeper into the script. I want to get rid of that here now. Also, don't know if we need here such generall functions but I think it's topic to discuss